### PR TITLE
LPD-34680 Ability to see all Refunds in the Order Details - part 2

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/LPD-34680

must be forwarded after this https://github.com/liferay-commerce/liferay-portal/pull/5330

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentEntryDisplayContext.java [hunk 1]
```diff
@@ -346,6 +348,24 @@ public String getRelatedToClassPK() {
 		return String.valueOf(_relatedCommercePaymentEntry.getClassPK());
 	}
 
+	public String getRelatedToLink() throws PortalException {
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentEntryDisplayContext.java:351]: Lets rename this to getRelatedToURL(), it aligns more with current naming patterns

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by dsitu** [comment]
@CrescenzoRegaLF Also did you see bchans comment [here](https://github.com/brianchandotcom/liferay-portal/pull/153853#issuecomment-2328845271).

Just gotta add these finders into service.xml for entity CommercePaymentEntry, so that the indexes will be created for better performance. Based of [this](https://github.com/brianchandotcom/liferay-portal/pull/153853/commits/756283b077c5729483bd74f4b077eb500a29a1e0#diff-0e1b0f10efac1cc5ce916625afd0e090111339ffa20a84cf05ce84316a8f221fR472-R495).


		<finder name="C_C_C_P_T" return-type="Collection">
			<finder-column name="companyId" />
			<finder-column name="classNameId" />
			<finder-column name="classPK" />
			<finder-column name="paymentStatus" />
			<finder-column name="type" />
		</finder>
		<finder name="C_T" return-type="Collection">
			<finder-column name="classPK" />
			<finder-column name="type" />
		</finder>

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
@dsitu can you help me understand why it's failing ?

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:forward

**Comment by CrescenzoRegaLF** [comment]
ci:forward:force

**Pull request closed without merging**